quakemigrate/io/data.py

```diff
@@ -608,6 +608,8 @@ def check_availability(
                     if len(st_id) > 1:
                         continue
                     elif st_id[0].stats.npts < n_samples:
+                        logging.debug("Trace has too few samples.")
+                        logging.debug((n_samples, st_id[0].stats.npts))
```

Comment:
Same here, just a little additional clarity:

```suggestion
                        logging.debug(f"(n_samples, trace_npts) : ({n_samples}, {st_id[0].stats.npts})")
```